src/_posts/languages/ruby/2000-01-01-getting-started-with-ruby.md

```diff
@@ -155,6 +155,23 @@ $ git push scalingo master
  <-- https://my-ruby-app.osc-fr1.scalingo.io -->
 ```
 
+## Database migrations
+
+To run a migration, you need to use a [post-deployment hook]({% post_url platform/app/2000-01-01-postdeploy-hook %}).
+
+The Ruby application starts up before the schema has changed, but does not receive a query.
+
+As a result, if operations are started before the migration has been executed, the application may load the schema `pre-migration`, and so a restart may be necessary after the migration, however short it may be.
```

Comment:
I find this sentence too long, hence hard to understand. Can you slice it in smaller pieces?

Reply:
@EtienneM Thanks to you about the review !

The content is not exactly the same, as there are specific mentions to Rails who aren't present in the Ruby page, but the note part is. I'll apply that to the note.